install/gcp/upgrade_tools/db_migrator.py

```diff
@@ -0,0 +1,113 @@
+import sys
+
+# Importing migrate.changeset adds some new methods to existing SQLAlchemy
+# objects but we will not be calling the library directly.
+import migrate.changeset  # noqa: F401, pylint: disable=unused-import
+from sqlalchemy.exc import OperationalError
+
+import google.cloud.forseti.services.scanner.dao as scanner_dao
+import google.cloud.forseti.services.inventory.storage as inventory_dao
+import google.cloud.forseti.services.dao as general_dao
+
+from google.cloud.forseti.common.util import logger
+
+
+DEFAULT_DB_CONN_STR = 'mysql://root@127.0.0.1:3306/forseti_security'
+LOGGER = logger.get_logger(__name__)
+
+
+class ColumnAction:
+    """Column action class."""
+    DROP = 'DROP'
+    CREATE = 'CREATE'
+
+
+def create_col(table, col):
+    """Create Column.
+
+    Args:
+        table (Table): The table object.
+        col (Column): The column object."""
```

Comment:
Can you be more specific on the `Column` and `Table` object's type here?  Other than just 'Column` and Tables.

Reply:
Updated.